gradle/build.gradle

```diff
@@ -17,7 +17,7 @@ plugins {
 }
 
 group 'com.flurry'
-version '4.2.0'
+version '4.2.1'
```

Comment:
`version '4.2.0'` was chosen to match the Flurry plugin version to the gradle version that we anticipated clients might be using (i.e. the Android Gradle Plugin (AGP) version)...

The latest AGP as of now is 7.3.0 (https://developer.android.com/studio/releases/gradle-plugin)

I think it might make more sense to our users if we continue to align with these AGP version numbers. Note: this has nothing to do with compatibility/correctness, and is really more about the optics of combining 7.x stuff with 4.x stuff.

Reply:
And a related question about https://github.com/flurry/upload-clients/pull/49/files#diff-67b4ee517b1cfa8159abb5acca24736745f13f87d03b9fe11fa7cd5f8a02cb44R31:
```
compileOnly 'com.android.tools.build:gradle:[4.2.0,5.0)'
```
Do we/can we build the Flurry plugin using a newer buildchain, or does that create issues for folks?

Reply:
Yes, I can publish the symbols with AGP 7.3.